styles/globals.css

```
@@ -56,8 +56,8 @@ form {
 /* Styles for the form labels */
 label {
   font-size: 16px;
-  font-family: "Lato", sans-serif;
-  color: #555;
+  font-family: "Inter", sans-serif;
```

```suggestion
  font-family: "Inter", "Lato", "Helvetica", sans-serif;
```
Nice having fallbacks